src/celengine/category.cpp

```diff
@@ -99,13 +99,13 @@ void UserCategory::cleanup()
     while(!m_objlist.empty())
     {
         auto it = m_objlist.begin();
-        DPRINTF(1, "Removing object: %s\n", it->getName());
+        DPRINTF(1, "Removing object: %s\n", it->getName().c_str());
```

Comment:
this is not required because #361 reimplemented DPRINTF using fmtlib, so now it's valid to use %s with std::string